modules/visual_script/editor/visual_script_property_selector.cpp

```diff
@@ -118,9 +118,11 @@ void VisualScriptPropertySelector::_notification(int p_what) {
 		case EditorSettings::NOTIFICATION_EDITOR_SETTINGS_CHANGED: {
 			_update_icons();
 		} break;
+
 		case NOTIFICATION_ENTER_TREE: {
 			connect("confirmed", callable_mp(this, &VisualScriptPropertySelector::_confirmed));
 		} break;
+	
```

Comment:
There seems to be trailing whitespace on this line that should be removed:
![image](https://user-images.githubusercontent.com/4701338/154307143-e6e6d08b-5afc-4921-81d3-c1950850a11d.png)